current/runtime/metadata.md

```diff
@@ -56,22 +88,33 @@ The response looks like following contents, but with much more information in `r
 }
 ```
 
-The hexadecimal string in the `result` field wraps the runtime metadata in SCALE format. Go to [Low-level Data Format](overview/low-level-data-format.md) page to learn how to decode different types and get more information about the specification and implementations.
-
-Our hex blob starts with a hard coded magic number `6d657461` which represents *meta* in plain text. The next piece of data shows the version number of the metadata, here we are using `04` to represent version 4. We already mentioned that runtime metadata is composed by available pallets. In our case we have 9 pallets. After shifting 9 in binary representation two bits to the left, we get `24` in hex to represent the length of the array.
+The hexadecimal string in the `result` field wraps the runtime metadata in 
+[SCALE](../advanced/codec) format.
 
-The remaining blob encodes the metadata of each pallet. Learning more about decoding different types in the [struct](https://substrate.dev/rustdocs/master/frame_metadata/struct.ModuleMetadata.html), please refer to the [reference docs](https://substrate.dev/rustdocs/master/frame_metadata/index.html) and [Low-level Data Format](overview/low-level-data-format.md) page.
+The hex blob starts with a hard-coded magic number `6d657461`, which represents "meta" in plain 
+text. The next piece of data shows the version number of the metadata, here we are using `04` to 
+represent version 4. We already mentioned that runtime metadata is composed of data from the 
+runtime's pallets. In our case we have 9 pallets. After shifting 9 in binary representation two 
```

Comment:
@danforbes brought up that this is incorrect, since the metadata in this version has 31 pallets

The correct Request/Response for this is actually:
{
  "id": 1,
  "jsonrpc": "2.0",
  "method": "state_getMetadata",
  "params": ["0xca15c2f1e1540517697b6b5f2cc6bc0c60876a1a1af604269b7215970798bbed"]
}
{"jsonrpc":"2.0","result":"0x6d6574610b7c1853797374656d011853797374656d3c1c4163636f756e740...7473","id":1}
where `0b` is 11 and '7c' is the number of pallets after bit-shifting to the right twice

Reply:
Also the '0x' prefix is actually required when querying for "state_getMetadata"

Reply:
Thank you, @insipx!